### PR TITLE
Add basic CI, K8s manifests, Skaffold, enable service account in go

### LINKED ISSUE #11: Frontendservice - containerize for GKE

### PR DESCRIPTION
**Pull request by bharathkkb**
Fixes #11, #12, #13

- Added Cloudbuild
- Added Skaffold
- Added multi stage Dockerfile for Frontend
- Enabled cached dependencies in Dockerfile for Go
- Enables use of service account file in Go
- Added k8s manifests with secrets and env via ConfigMaps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Retail-Banking-Microservices-Demo
 
+## Quickstart
+
+1. Install requirements
+   - skaffold ≥v1.0.1
+   - kubectl
+   - Docker
```
Comment (meganzhao) [inline README.md]: Add installation links: 
https://skaffold.dev/docs/install/
https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos
https://docs.docker.com/v17.12/install/

File: README.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+2. Generate kubeconfig for your kubernetes cluster. The services will be deployed to this cluster
+   - For local development you can use k8s with Docker Desktop
+   - You can also configure kubeconfig for a remote cluster. With gcloud, `gcloud container clusters get-credentials [$CLUSTER_NAME] --project [$PROJECT_NAME] --[region/zone] [$REGION/ZONE]`
```
Comment (meganzhao) [inline README.md]: gcloud container clusters create [$CLUSTER_NAME] --zone [region/zone]

Comment (bharathkkb) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (bharathkkb) [reply README.md]: Done

**Pull request merged**